code/game/objects/items/devices/lighting/flare.dm

```diff
@@ -2,9 +2,9 @@
 	name = "flare"
 	desc = "A red standard-issue flare. There are instructions on the side reading 'twist cap off, make light'."
 	desc_info = "Use this item in your hand, to turn on the light."
-	w_class = WEIGHT_CLASS_SMALL
-	brightness_on = 3 // Pretty bright.
-	light_power = 4
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
Don't buff the old flares, they're used for shitty emergency kits.

Make a new variant.